.travis.yml

```diff
@@ -0,0 +1,75 @@
+shared_deployment_options: &shared
```

Comment:
is `&shared` valid?

Reply:
Yes, it's a way to not repeat all the keys:
https://web.archive.org/web/20130213112648/http://blog.101ideas.cz/posts/dry-your-yaml-files.html